res/css/views/rooms/_BasicMessageComposer.scss

```diff
@@ -45,22 +45,19 @@ limitations under the License.
         overflow-x: hidden;
 
         span.mx_UserPill, span.mx_RoomPill {
-            padding-left: 21px;
             position: relative;
 
             // avatar psuedo element
             &::before {
-                position: absolute;
-                left: 2px;
-                top: 2px;
                 content: var(--avatar-letter);
-                width: 16px;
-                height: 16px;
+                width: $font-16px;
+                height: $font-16px;
+                margin-right: 0.24rem;
```

Comment:
Not sure where this margin comes from ... Looking at the adhoc build, the avatar also has a border around it where it didn't before:
On ad-hoc:
![image](https://user-images.githubusercontent.com/274386/79005470-6fb0b400-7b46-11ea-8e66-78a0f6be38c9.png)
On develop:
![image](https://user-images.githubusercontent.com/274386/79005581-b30b2280-7b46-11ea-9759-9883dcb720d1.png)

Reply:
The previous mentions do have a border around them. I've just accidentily made them a little bigger than they should be. I'll fix it.

Reply:
The margin comes in as a result of replacing a large right padding used by the pill.

Reply:
The border is now much smaller.
<img width="948" alt="image" src="https://user-images.githubusercontent.com/23084468/79250065-14e4c880-7e76-11ea-8058-f637c39faab7.png">

I thought the border was intentional because of the following pills on develop:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/23084468/79250178-38a80e80-7e76-11ea-9988-9fb0ea4b3cc9.png">

I can get rid of the border on the message userpills altogether if it looks better.

Reply:
Ah, yeah, the pills do look a bit bigger in the auto-complete which I believe your screenshot was taking from?
It's a bit hard to see on your screenshot, but it looks like the padding-right is a bit smaller there, as in the text is more crammed against the end of the pill?